packages/editor-ui/src/components/MainHeader/CollaborationPane.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import { COLLABORATION_HEARTBEAT_INTERVAL } from '@/constants';
+import { useUsersStore } from '@/stores/users.store';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { useCollaborationStore } from '@/stores/collaboration.store';
+import { onBeforeUnmount } from 'vue';
+import { onMounted } from 'vue';
+import { computed, ref } from 'vue';
+
+const collaborationStore = useCollaborationStore();
+const usersStore = useUsersStore();
+const workflowsStore = useWorkflowsStore();
+
+const heartbeatInterval = COLLABORATION_HEARTBEAT_INTERVAL;
+const heartbeatTimer = ref(null as null | NodeJS.Timeout);
+
+const activeUsers = computed(() => {
+	return {
+		default: (collaborationStore.getUsersForCurrentWorkflow || []).map((userInfo) => userInfo.user),
+	};
```

Comment:
Why a object with `default` instead of just the array?

Also we should prefer `??` for null & undefined coalescing, since it then you don't need think of the complexity of type coercion that `||` does

```suggestion
	return {
		default: (collaborationStore.getUsersForCurrentWorkflow ?? []).map((userInfo) => userInfo.user),
	};
```

Reply:
> Why a object with `default` instead of just the array?
> 
> Also we should prefer `??` for null & undefined coalescing, since it then you don't need think of the complexity of type coercion that `||` does

The `UserStack` component supports user groups, not just flat lists (and hides group info if there is only one grup). So this is the name of the group. Will rename to something else since this seems confusing.